addons/account/wizard/account_move_reversal.py

```diff
@@ -14,21 +14,16 @@ class AccountMoveReversal(models.TransientModel):
 
     move_ids = fields.Many2many('account.move', 'account_move_reversal_move', 'reversal_id', 'move_id', domain=[('state', '=', 'posted')])
     new_move_ids = fields.Many2many('account.move', 'account_move_reversal_new_move', 'reversal_id', 'new_move_id')
-    date_mode = fields.Selection(selection=[
-            ('custom', 'Specific'),
-            ('entry', 'Journal Entry Date')
-    ], required=True, default='custom')
     date = fields.Date(string='Reversal date', default=fields.Date.context_today)
-    reason = fields.Char(string='Reason')
+    reason = fields.Char(string='Reason displayed on Credit Note')
     refund_method = fields.Selection(selection=[
             ('refund', 'Partial Refund'),
-            ('cancel', 'Full Refund'),
             ('modify', 'Full refund and new draft invoice')
         ], string='Credit Method', required=True,
         help='Choose how you want to credit this invoice. You cannot "modify" nor "cancel" if the invoice is already reconciled.')
```

Comment:
This field is no longer used in the wizard, so we should remove it :broom: :slightly_smiling_face: